lib/ansible/module_utils/alicloud_ecs.py

```diff
@@ -0,0 +1,176 @@
+#
```

Comment:
Hi @sivel ,

Thank you for your feedback. I noticed that other cloud module_utils don't carry a BSD license, and why this module need to do it? 
Looking forward to your feedback, thanks.

Regards,
Guimin